samples/snippets/src/test/java/com/example/spanner/SetMaxCommitDelaySampleIT.java

```diff
@@ -0,0 +1,106 @@
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.KeySet;
+import com.google.cloud.spanner.Mutation;
+import com.google.common.collect.ImmutableList;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collections;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Integration tests for {@link SetMaxCommitDelaySample}
+ */
+@RunWith(JUnit4.class)
+public class SetMaxCommitDelaySampleIT extends SampleTestBase {
+
+  private static DatabaseId databaseId;
+
+  @BeforeClass
+  public static void createTestDatabase() throws Exception {
+    final String database = idGenerator.generateDatabaseId();
+    databaseAdminClient
+        .createDatabase(
+            instanceId,
+            database,
+            ImmutableList.of(
+                "CREATE TABLE Singers ("
+                    + "  SingerId   INT64 NOT NULL,"
+                    + "  FirstName  STRING(1024),"
+                    + "  LastName   STRING(1024),"
+                    + "  SingerInfo BYTES(MAX)"
+                    + ") PRIMARY KEY (SingerId)",
+                "CREATE TABLE Albums ("
+                    + "  SingerId        INT64 NOT NULL,"
+                    + "  AlbumId         INT64 NOT NULL,"
+                    + "  AlbumTitle      STRING(MAX),"
+                    + "  MarketingBudget INT64"
+                    + ") PRIMARY KEY (SingerId, AlbumId),"
+                    + "  INTERLEAVE IN PARENT Singers ON DELETE CASCADE"))
+        .get();
+    databaseId = DatabaseId.of(projectId, instanceId, database);
+  }
+
+  @Before
+  public void insertTestData() {
+    final DatabaseClient client = spanner.getDatabaseClient(databaseId);
+    client.write(Arrays.asList(
+        Mutation.newInsertBuilder("Singers")
+            .set("SingerId")
+            .to(1L)
+            .set("FirstName")
+            .to("first name 1")
+            .set("LastName")
+            .to("last name 1")
+            .build(),
+        Mutation.newInsertBuilder("Singers")
+            .set("SingerId")
+            .to(2L)
+            .set("FirstName")
+            .to("first name 2")
+            .set("LastName")
+            .to("last name 2")
+            .build()
+    ));
+  }
+
+  @After
+  public void removeTestData() {
+    final DatabaseClient client = spanner.getDatabaseClient(databaseId);
+    client.write(Collections.singletonList(Mutation.delete("Singers", KeySet.all())));
+  }
+
+  @Test
+  public void testSetMaxCommitDelaySample() throws Exception {
+    final DatabaseClient client = spanner.getDatabaseClient(databaseId);
+    final String out = runSample(() -> SetMaxCommitDelaySample.setMaxCommitDelay(client));
+
+    assertThat(out).contains("Updated data");
```

Comment:
@nginsberg-google Have we verified that this test executes successfully? The samples tests are not part of pre-submits (since they are very slow). They run as post submits in continuous test.

Reply:
I don't know how to. I am not able to run this from IntelliJ, and couldn't figure out how to run it from the command line. Can you please either tell me how to run it and I will, or would you be willing to pull this locally and run it? Whatever is easiest for you.

Reply:
Verified locally that the test succeeds.